ssg/boolean_expression.py

```diff
@@ -93,6 +93,10 @@ def arg(self):
     def name(self):
         return self.requirement.project_name
 
+    @staticmethod
+    def reference_is_parametrized(reference):
```

Comment:
The problem with naming still remains - when considering that `Symbol` class, what is supposed to be this `reference` argument that the method accepts?